lib/ansiblelint/utils.py

```diff
@@ -276,9 +276,13 @@ def append_children(taskhandler, basedir, k, parent_type, results):
 
 def _roles_children(basedir, k, v, parent_type, main='main'):
     results = []
+    print v
```

Comment:
This is python2-only syntax which is completely incompatible with modern Python interpreter versions. Use print-function instead of a statement if you really need it.